### PR TITLE
Enable the previewed layer

### LINKED ISSUE #491: Enable the previewed layer
When we preview a layer that is a non-enabled visualisation set, the layer is not displayed.

The user needs to activate the visualisation manually in order to display the layer

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**
Fix #491

### CODE REVIEW

File: src/app/services/main-form-manager/config-export-helper.ts [hunk 1]
```diff
@@ -313,7 +314,7 @@ export class ConfigExportHelper {
             mapConfigVisualisations.controls.forEach(visu => visualisationsSets.push({
                 name: visu.value.name,
                 layers: visu.value.layers,
-                enabled: visu.value.displayed
+                enabled: (enableByDefault === true) ? true : visu.value.displayed
```
Comment (sebbousquet) [inline src/app/services/main-form-manager/config-export-helper.ts:317]:
```suggestion
                enabled: enableByDefault ? true : visu.value.displayed
```

File: src/app/services/main-form-manager/config-export-helper.ts [hunk 2]
```diff
@@ -323,7 +324,8 @@ export class ConfigExportHelper {
                     visualisationsSets.push({
                         name: visu.value.name,
                         layers: [arlasId],
-                        enabled: visu.value.displayed
+                        // this will activate the visualisation set and display its layers for preview purposes
+                        enabled: (enableByDefault === true) ? true : visu.value.displayed
```
Comment (sebbousquet) [inline src/app/services/main-form-manager/config-export-helper.ts:328]:
```suggestion
                        enabled: enableByDefault ? true : visu.value.displayed
```

**Review by sebbousquet — CHANGES_REQUESTED** [review]

File: src/app/services/main-form-manager/config-export-helper.ts (hunk 1 above)
Comment (MohamedHamouGisaia) [reply src/app/services/main-form-manager/config-export-helper.ts]: fixed

File: src/app/services/main-form-manager/config-export-helper.ts (hunk 2 above)
Comment (MohamedHamouGisaia) [reply src/app/services/main-form-manager/config-export-helper.ts]: fixed

**Review by sebbousquet — APPROVED** [review]

**Review by mbarbet — APPROVED** [review]

**Pull request merged**